docs/api.rst

```diff
@@ -3,3 +3,9 @@ Reference/API
 
 .. automodapi:: regions
    :no-inheritance-diagram:
+
+.. automodapi:: regions.io.ds9
+   :no-inheritance-diagram:
+
+.. automodapi:: regions.io.stcs
+   :no-inheritance-diagram:
```

Comment:
Probably should reject this change - no reason to exclude inheritance diagrams, right?